js/packages/rnutil/checkPermissions.ts

```diff
@@ -61,6 +67,11 @@ export const requestPermission = async (
 	if (permissionType === 'notification') {
 		return (await requestNotifications(['alert', 'sound'])).status
 	}
+	if (permissionType === 'camera') {
```

Comment:
First, please use the enum that we create for the string type 👍 
This condition is only for web, so 2 solutions:
* Change the current condition
```js
if (permissionType === PermissionType.camera && Platform.OS === 'web') {
	...blabla
}
```
* Or in the entry of the function, uncomment the check of the platform and add this
```js
if (Platform.OS === 'web' && permissionType !== PermissionType.camera) {
	return RESULTS.DENIED
}
```

Soon we'll refacto that so no problem if it's a little confusing..